tox.ini

```diff
@@ -90,6 +92,13 @@ commands =
     pytest ./lib/pbench/test/unit/common
     pytest ./lib/pbench/test/unit/server 
     pytest ./lib/pbench/test/functional
+    pytest --cov --cov-append \
+        --cov-report html:cov_html \
+        --cov-report xml:cov.xml \
+        --cov-report annotate:cov_annotated \
```

Comment:
This might be too much, sorry for suggesting it.  I adds `.cov` files to everything ...